src/actions/PatientActions.js

```diff
@@ -67,6 +67,7 @@ const patientUpdate = patientDetails => (dispatch, getState) => {
     addressOne: patientLine1,
     addressTwo: patientLine2,
     country: patientCountry,
+    supplyingStoreId: patientSupplyingStoreId,
```

Comment:
ahh ok I think I see whatsup!

The patient form isn't already disabled for patients being fetched from the API?

I Think that's because the default in `createRecord` for a patient is that the field `thisStoresPatient` is being set as true and we're not overriding it for a non-local patient (same for a prescriber?).

I think if we change this that should be all that is needed